### PR TITLE
Feature/ui styling adapter modals ff safari

### PR DESCRIPTION
**Pull request by astewes**
*  Adds custom styling for select form element
*  Adds helper class to facilitate custom styling
*  Changes some hex color values to variables
*  Contains some minor less code cleanup

Below are GIFs of the adapter modal select elements in action:

Modal header buttons using % width:
ff2

Firefox: 
firefox

Safari:
safari

Chrome:
chrome

### CODE REVIEW

File: cdap-ui/app/features/adapters/etlapps.less [hunk 1]
```diff
@@ -193,7 +191,7 @@ body.theme-cdap.state-adapters-create {
         .modal-btns {
           margin-right: -15px;
           text-align: right;
-
+          height: 60px;
```
Comment (ajainarayanan) [inline cdap-ui/app/features/adapters/etlapps.less]: We might not need a height here. The individual button widths need to be managed with media queries.

File: cdap-ui/app/styles/themes/cdap/mixins.less [hunk 2]
```diff
@@ -154,3 +154,13 @@
   order: @val;
 }
 
+.appearance(@value: none) {
+  appearance: @value;
+  -webkit-appearance: @value;
+  -moz-appearance: @value;
+  -o-appearance: @value;
+  ::-ms-expand {
+    display: none;  // Need this to hide default select in IE10
```
Comment (ajainarayanan) [inline cdap-ui/app/styles/themes/cdap/mixins.less]: If we hide this for IE10 would we be even seeing the dropdowns in that browser?

**Comment by astewes** [comment]
in IE10, the default dropdown arrow would be hidden, but the custom arrow (placed via the helper class .select-wrapper) would take care of this

**Comment by ajainarayanan** [comment]
:+1: LGTM

**Pull request merged**